src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/SetNodeIdTypeMessageClass.java

```diff
@@ -30,12 +30,15 @@
 public class SetNodeIdTypeMessageClass extends ZWaveCommandProcessor {
     private final Logger logger = LoggerFactory.getLogger(SetNodeIdTypeMessageClass.class);
 
+    private static byte cmdSetNodeIdType = (byte) 0x80; // Subclass of SetUpZwaveAPI for NodeID Type
+    private static byte nodeIdType8Bit = (byte) 0x01;  // 1=8 bit, 2=16 bit NodeID Type
```

Comment:
Sorry to be a pain, but please make these `final`. Also, as per my suggestion, please use the convention of upper case for constants, delimited with underscores rather than camel case.

Reply:
Thanks, and no worries - I appreciate the changes and sorry to be picky :)